### ISSUE TITLE
A simple function automatically collapse the node the next time you open the interface

### ISSUE THREAD

**Issue by CandyTek** [giuspen/cherrytree#2514]
I want a simple function that can be added as a switch in the settings:

Collapse all tree nodes on the left side each time the interface is opened from the tray.

This will facilitate the recording of each opening, eliminating the need to manually collapse the nodes with the mouse or use shortcut keys the next time you open it.

**Comment by giuspen** [comment]
This option already exists in the preferences dialog, tab 'Tree Explorer'

![image](https://github.com/giuspen/cherrytree/assets/6842059/1e1d0cf6-9781-46c6-820b-583c01c22bdf)

**Comment by CandyTek** [comment]
Yes, there is a preference similar to what I described in the settings, which shrinks all trees every time the user cold starts (starts the program from scratch).
But most of the time when using it, CherryTree is hidden in the background. When recording code, click the tray or global shortcut key to call out the window. At this time, this preference does not work in this scenario.

Thanks to the author and contributors for making such an excellent software, it helps me a lot. I use it to record some uncommon codes, which can speed up my code writing. Cheers!
